mesonbuild/modules/gnome.py

```diff
@@ -602,13 +601,13 @@ def yelp(self, state, args, kwargs):
 
         script = [sys.executable, state.environment.get_build_command()]
         args = ['--internal',
-            'yelphelper',
-            'install',
-            '--subdir=' + state.subdir,
-            '--id=' + project_id,
-            '--installdir=' + os.path.join(state.environment.get_datadir(), 'help'),
-            '--sources=' + source_str,
-        ]
+                'yelphelper',
+                'install',
+                '--subdir=' + state.subdir,
+                '--id=' + project_id,
+                '--installdir=' + os.path.join(state.environment.get_datadir(), 'help'),
+                '--sources=' + source_str,
+               ]
```

Comment:
__[flake8]__
* [E124] closing bracket does not match visual indentation

<!-- Comment by [SideCI](https://sideci.com) -->